generate/template/astronomy.py

```diff
@@ -5508,7 +5512,7 @@ def _ToggleAzimuthDirection(az):
     return az
 
 
-def VectorFromHorizon(sphere, time, refraction):
+def VectorFromHorizon(sphere: Spherical, time: Time, refraction: Refraction):
```

Comment:
This one returns Vector. Do you omit the type hint for the return value for the ones where it is obvious?

Reply:
Ah no, sorry I missed this one as well.